3-JSS/src/webpackTodoList.js

```diff
@@ -0,0 +1,148 @@
+import {
```

Comment:
will it be better if you separate this file into smaller logical files?

Reply:
True. Ill take the "Create" elements function to a util file, and the rest will leave at the app file I think.